content/embeds/rc-opt-in-to-72.md

```diff
@@ -0,0 +1,7 @@
+Select **Opt-in to 7.2** to upgrade your subscription and databases to Redis 7.2 (*Redis 6.2 and earlier only*).
+
+{{<image filename="images/rc/button-opt-in-to-72.png" width=150px alt="Opt-in to 7.2 button." >}}{{< /image >}}
+
+The upgrade will occur during low-traffic hours or during a manually set [maintenance window]({{<relref "/rc/subscriptions/maintenance/set-maintenance-windows">}}).
```

Comment:
The upgrade will occur according to the Maintenance Windows of the subscription (no need to write low-traffic hours.. it may confuse)

Reply:
And let's add something regarding the fact that we will upgrade the subscription in a week from the request.